publication-commons/src/main/java/no/unit/nva/publication/service/impl/UpdateResourceService.java

```diff
@@ -163,7 +163,6 @@ public void updateOwner(SortableIdentifier identifier, UserInstance oldOwner, Us
     }
 
     public void updateResource(Resource resource) {
-        resource.setModifiedDate(clockForTimestamps.instant());
```

Comment:
a bit scare to change this maybe?

Reply:
we have at least protect against new users not remembering to setModifiedDate when they should

Reply:
Strong agreement here. It's the responsibility of Resource to set its own modification time